### DISCUSSION TITLE
How to disable code block invokation for indented content?

### DISCUSSION THREAD

**Discussion by ivanjaros** [markedjs/marked#3420] [Q&A]
I want to use only three backticks for code blocks, i do not want to use indentation to trigger code block. This seems to be native markdown behavior and is not tied to GFM or anything like that. Is there some way to disable this?

**Comment by UziTech** [comment]
You can disable the `code` tokenizer to prevent indented code. The ` ``` ` code is tokenized by the `fences` tokenizer

```js
marked.use({
  tokenizer: {
    code() {
      // return undefined to disable
    }
  }
});
```
